pkg/solana/client/multinode/config/config.go

```diff
@@ -103,7 +105,7 @@ func (c *MultiNodeConfig) SetDefaults() {
 	}
 	// Selection mode defaults to priority level to enable using node priorities
```

Comment:
These defaults are only applicable to Solana. They should not be stored in generalized multinode pkg